pkg/vulnerabilityreport/controller.go

```diff
@@ -4,10 +4,10 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/aquasecurity/trivy-operator/pkg/operator/workload"
```

Comment:
can we revert this file change?

Reply:
Sure I can and did, although running a standard go fmt linter over this file will re-arrange them as per alphabetical order again as it isn't so today.